FrontEnd/src/components/HeaderFooter/header/navbar/Profile.jsx

```diff
@@ -1,12 +1,14 @@
-import css from "./Profile.module.css"
-import avatar_image from "./Avatar.png"
+import css from "./Profile.module.css";
+import avatar_image from "./Avatar.png";
+import { Link } from 'react-router-dom';
 
 function Profile(props) {
     return (
         <div className={css["header-profile-section"]}>
             <img className={css["header-profile__avatar"]} src={avatar_image} alt=""/>
             <div className={css["header-profile-tab"]}>
-                <a className={css["header-profile-link__text"]} href="profile">Профіль</a>
+                {/* <a className={css["header-profile-link__text"]} href="profile">Профіль</a> */}
```

Comment:
Do we still need it?

Reply:
no

Reply:
Thanks for your comment. It was fixed in a new commit